### ISSUE TITLE
How to specify using Independent Graphic card?

### ISSUE THREAD

**Issue by whisperzh** [GestureGeneration/Speech_driven_gesture_generation_with_autoencoder#5]
I successfully get the samples trained, but it seems that i was using my intergrated graphic card.  Could you help me

**Comment by Svito-zar** [comment]
I assume that your problem is that the training was not utilizing your GPU.

The following aspects are important to ensure that you use your GPU:

1.  Install Tensorflow which supports GPU:
`pip install tensorflow-gpu==1.14.0`

2.  Specify the GPU to be used during the training:
`CUDA_VISIBLE_DEVICES=1 python train.py MODEL_NAME EPOCHS DATA_DIR N_INPUT ENCODE DIM` 

Hope that helps.
If it does not - please provide more details about your issue.

**Comment by whisperzh** [comment]
OK, i understand how to do, THANK U for helping me. And i am still dealing with my environment, since i doesn't have CUDA. I'll let you know as soon as it works . Again Thank you!!

**Comment by Svito-zar** [comment]
I close this issue due to inactivity
